mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -266,11 +266,11 @@ def process_data(test=False):
 
     create_netflix_csv(netflix_csv, processed_data)
 
-    print(f"missing:  {missing_count} ({missing_count / num_rows * 100:.2f}%)")
     print(
-        f"found: {num_rows - missing_count} ({(num_rows - missing_count) / num_rows * 100:.2f}%)"
+        f"missing: {missing_count} ({missing_count / num_rows * 100:.2f}%)\n"
```

Comment:
I don't think merging this into one print statement with `\n` separating them makes it any more readable. There's already a readability challenge in the extensive use of `{}` notation. It is easy to reason that each `print` statement goes on its own line, but harder to notice the `\n` characters in this combination.

Reply:
I did it just cause it makes the whitespace more consistent so it looks in the code how it does in the output. I can revert this if you still disagree.

Reply:
Your choice, just offering my opinion.